Lib/test/test_asyncio/test_events.py

```diff
@@ -2782,6 +2783,8 @@ async def main():
                 self.loop.run_until_complete(main()),
                 'hello')
 
+            multiprocessing_cleanup_tests()
```

Comment:
It looks like this won't clean up if there's a test failure. Couldn't you use `self.addCleanup(multiprocessing_cleanup_tests)` before the `assertEqual()` call?

Reply:
Ok, done.